op-e2e/e2eutils/setup.go

```diff
@@ -207,7 +208,6 @@ func Setup(t require.TestingT, deployParams *DeployParams, alloc *AllocParams) *
 		ChannelTimeout:         deployConf.ChannelTimeout,
 		L1ChainID:              new(big.Int).SetUint64(deployConf.L1ChainID),
 		L2ChainID:              new(big.Int).SetUint64(deployConf.L2ChainID),
-		P2PSequencerAddress:    deployConf.P2PSequencerAddress,
```

Comment:
We need to consider backwards compatibility here: if we remove the ability to configure this, then we will need to upgrade the proxy of the `SystemConfig` contract on the L1 networks that are already running

Reply:
If you're referring to the beta, I wouldn't worry about it. We've already introduced backwards incompatible changes to the beta.